src/lib/import-export/export/exporters/default-exporter.ts

```diff
@@ -165,7 +165,7 @@ export class DefaultExporter extends EventEmitter implements Exporter {
 			const tmpFolder = await fsPromises.mkdtemp( path.join( os.tmpdir(), 'studio_export' ) );
 			const fileName = `${ generateBackupFilename( 'db-export' ) }.sql`;
 			const sqlDumpPath = path.join( tmpFolder, fileName );
-			await exportDatabaseToFile( this.options.site, sqlDumpPath );
+			await exportDatabaseToMultipleFiles( this.options.site, sqlDumpPath );
```

Comment:
When we continue working on this draft, let's plan to make this change only to export file produced for Sync purposes, leaving Export as is.

Reply:
I think it's easier and simpler to modify the export full site too.
If a user wants to to create a single SQL file, then we have the export database button, which is easier and faster than exporting the full site and unzipping it.

Any big reason to do not follow the same patter for export site? I see it as an improvement and in the future users could use it to restore their backup manually to other Jetpack Site.

Reply:
From my quick assessment, it seems like this would work out of the box with our existing importers, right..?

Reply:
@sejas one reason is to limit the number of changes we make at once, so we avoid updating the Export feature when we work on the Sync feature.

Another one is that I think we should work on fixing VaultPress import to accept a full dump at some point and consider the current change a temporary solution. A package that includes the whole database in one file is a more common approach and can be easily created by the user to prepare a site export compatible with Studio.